tests/scf-response3/input.dat

```diff
@@ -0,0 +1,30 @@
+# UHF Dipole Polarizability Test
+
+molecule {
+1 2
+H    0.7493682    0.1000000    0.4424329
+O    0.0000000    0.2000000   -0.1653507
+H   -0.7493682    0.0000000    0.4424329
+no_com
+no_reorient
+symmetry c1
+}
+
+set reference uhf
+set scf_type pk
+
+psi4.properties("scf/cc-pVDZ", properties=["DIPOLE_POLARIZABILITIES"])
+
+alpha_ref = [
```

Comment:
Where does the reference come from?

Reply:
The program that should not be named... 😄 I will add a comment to the input file.

Reply:
Done.